client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx

```diff
@@ -557,33 +558,17 @@ const InvestigationTable: React.FC = (): JSX.Element => {
                         </Button>
                     </Box>
                 </div>
-                <Grid container justify="flex-end" className={classes.filterTableRow}>
-                    <Collapse in={showFilterRow}>
-                        <Card className={classes.filterTableCard}>
-                            <Typography>סינון לפי</Typography>
-                            <Typography>סטטוס:</Typography>
-                            <Autocomplete
-                                classes={{ inputRoot: classes.autocompleteInput }}
-                                disableCloseOnSelect
-                                multiple
-                                options={allStatuses}
-                                value={allStatuses.filter(status => statusFilter.includes(status.id))}
-                                getOptionLabel={(option) => option.displayName}
-                                onChange={onSelectedStatusesChange}
-                                renderInput={(params) =>
-                                    <TextField
-                                        {...params}
-                                    />
-                                }
-                                renderTags={(tags) => {
-                                    const additionalTagsAmount = tags.length - 1;
-                                    const additionalDisplay = additionalTagsAmount > 0 ? ` (+${additionalTagsAmount})` : '';
-                                    return tags[0].displayName + additionalDisplay;
-                                }}
-                            />
-                            <IconButton onClick={() => closeFilterRow()}><Close /></IconButton>
-                        </Card>
-                    </Collapse>
+                <Grid container justify='flex-end' alignItems='center' className={classes.filterTableRow}>
+                    <Grid item xs={12} md={4}>
+                        <Collapse in={showFilterRow}>
+                            <StatusFilter
+                                statuses={allStatuses}
+                                filteredStatuses={statusFilter}
+                                onStatusChange={onSelectedStatusesChange}
+                                onClose={closeFilterRow}
+                                />
```

Comment:
god bless you for the refactoring! fix last line indentation

Reply:
❤️ fixed